app/controllers/admin/base_controller.rb

```diff
@@ -1,3 +1,9 @@
 class Admin::BaseController < ApplicationController
```

Comment:
Good idea to have this method in an own controller. 
But it would have been "cleaner" to have it in a seperate pull request since it has nothing to do with the translation (same for the rake tasks).